neo/Trie/MPT/MPTNode/LeafNode.cs

```diff
@@ -22,12 +22,12 @@ public LeafNode(byte[] val)
 
         public override void EncodeSpecific(BinaryWriter writer)
         {
-            writer.WriteVarBytes(Value);
+            writer.WriteBytesWithGrouping(Value);
         }
 
         public override void DecodeSpecific(BinaryReader reader)
         {
-            Value = reader.ReadVarBytes(ushort.MaxValue);
+            Value = reader.ReadBytesWithGrouping();
```

Comment:
There are no max?

Reply:
We don't have max in storage value.
If we have max  here,  the state will be different from storages.

Reply:
You can use this https://github.com/neo-project/neo-vm/blob/30e9d573d1fcad40a21f6ee14d7eb9703a8a6d10/src/neo-vm/ExecutionEngine.cs#L40